akka-http-core/src/test/scala/akka/http/impl/engine/client/HttpsProxyGraphStageSpec.scala

```diff
@@ -4,27 +4,60 @@
 
 package akka.http.impl.engine.client
 
+import akka.http.impl.util.EnhancedString
+import akka.http.scaladsl.model.headers.{ BasicHttpCredentials, HttpCredentials, `Proxy-Authorization` }
 import akka.http.scaladsl.settings.ClientConnectionSettings
-import akka.stream.scaladsl.{ BidiFlow, Flow, Keep, Sink, Source }
-import akka.stream.testkit.{ TestPublisher, TestSubscriber, Utils }
+import akka.stream.scaladsl.{ Flow, Keep, Sink, Source }
 import akka.stream.testkit.scaladsl.{ TestSink, TestSource }
+import akka.stream.testkit.{ TestPublisher, TestSubscriber, Utils }
 import akka.stream.{ ActorMaterializer, ActorMaterializerSettings }
 import akka.testkit.AkkaSpec
 import akka.util.ByteString
-
 import scala.concurrent.duration._
 
 class HttpsProxyGraphStageSpec extends AkkaSpec {
   implicit val materializer = ActorMaterializer(ActorMaterializerSettings(system).withFuzzing(true))
 
   "A ProxyGraphStage" should {
     "send CONNECT message and then forward incoming messages" in new Context {
-      testCase { (source, flowInProbe, flowOutProbe, sink) ⇒
+      testCase() { (source, flowInProbe, flowOutProbe, sink) ⇒
+        source.sendNext(ByteString("anything"))
+        sink.request(1)
+
+        val connectMsg = flowInProbe.requestNext()
+        connectMsg.utf8String should startWith(s"CONNECT $targetHostName:$targetPort HTTP/1.1")
+        flowOutProbe.sendNext(ByteString("HTTP/1.0 200 Connection established\r\n\r\n"))
```

Comment:
From what I've seen just now "in the wild" many http proxies seem to reply using 1.0 weirdly enough... 
Not that it matters in the test though, can stay as is